lib/src/Base/Algo/KDTree.cxx

```diff
@@ -25,13 +25,25 @@
 #include "openturns/Indices.hxx"
 #include "openturns/SobolSequence.hxx"
 #include "openturns/PersistentObjectFactory.hxx"
+#include "openturns/ResourceMap.hxx"
+#include "openturns/OTconfig.hxx"
+
+#ifdef OPENTURNS_HAVE_NANOFLANN
+#include <nanoflann.hpp>
+#if NANOFLANN_VERSION < 0x150
```

Comment:
Do we need to support older versions of nanoflann? Maybe due to packaging in some distributions?

Reply:
we, it can be older in debian/ubuntu